cmd/manager/main.go

```diff
@@ -79,11 +84,15 @@ func main() {
 
 	ctx := context.TODO()
 
-	// Become the leader before proceeding
-	err = leader.Become(ctx, "integreatly-operator-lock")
-	if err != nil {
-		log.Error(err, "")
-		os.Exit(1)
+	if isLeaderElectionEnabled() {
+		// Become the leader before proceeding
+		err = leader.Become(ctx, "integreatly-operator-lock")
+		if err != nil {
+			log.Error(err, "")
+			os.Exit(1)
+		}
+	} else {
+		log.Info("Warning: Leader election is disabled")
 	}
```

Comment:
This is required for CI. To become a leader it needs to be running in a cluster, and would skip it if being run locally like we do in CI tests. Since the CI is running on a kubernetes cluster though, this thinks it is running in a cluster and fails the tests. This just allows us to disable it when executing it locally in CI.